build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -313,6 +313,8 @@ CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="60,40,12,400,1844,4808,384,1844,4808,384,512,1280,8,"
 CONFIG_FLASH_PART_TYPE="none,none,none,none,kernel,bin,bin,kernel,bin,bin,smartfs,ftl,bootparam,"
 CONFIG_FLASH_PART_NAME="bl1,reserved,ftl,ss,kernel,app1,app2,kernel,app1,app2,userfs,reserved,bootparam,"
+CONFIG_TRPK_CONTAINS_MULTIPLE_BINARY=y
+CONFIG_FLASH_VSTART_LOADABLE=0xe000000
```

Comment:
I think this defconfig does not enable `XIP_ELF` which you make a dependency. 
@abhishek-samsung Could you guide him which defconfig should be enabled?

Reply:
But in https://github.com/Samsung/TizenRT/pull/6185, the loadable configs will apply XIP_ELF.
Also as I have known, there seems no more loadable ram cases for RTL8730E in the future, due to ram shortage.

Reply:
> @abhishek-samsung Could you guide him which defconfig should be enabled?

in the current code loadable_apps_xip/defconfig is the only one using xip apps. So changing only that particular config should be enough. I think #6185 can apply these changes required in that PR self.

Reply:
okay, then I will only update to loadable_app_xip/defconfig